vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -332,8 +333,7 @@ private void synchronizeChildComponentValues(LocalDateTime value) {
      */
     private void addToSlot(Component component, String slot) {
         Objects.requireNonNull(component, "Component to add cannot be null");
-        component.getElement().setAttribute("slot", slot);
-        getElement().appendChild(component.getElement());
+        SlotUtils.addToSlot(this, slot, component);
```

Comment:
**suggestion:** I guess this method can be removed in favor of direct `SlotUtils.addToSlot` invocations in the constructor. This non-null check doesn't seem necessary as, based on the source code, this method is only invoked with picker instances which cannot be null.

Reply:
Good point, thanks. Done.